### PR TITLE
MAINT: Linting

### PR DESCRIPTION
**Pull request by jakereps**
Fixes no open issues, just my sanity 🙃.

### CODE REVIEW

**Comment by jairideout** [comment]
What's the status of this PR? Is it worth linting at this point or waiting for the studio overhaul?

cc: @thermokarst @ebolyen

**Comment by jakereps** [comment]
It's done and it should be good to go. It's entirely lint issues and some tooling updates. Won't hurt anything either way.

**Comment by jairideout** [comment]
Thanks @jakereps! @thermokarst, @ebolyen, can one of you take the lead on this?

File: app/main.js [hunk 1]
```diff
@@ -49,7 +49,7 @@ const startRestAPI = (callback) => {
     api.stdout.on('data', (data) => {
         if (!started) {
             started = true;
-            callback.apply(null, data.toString('utf8').split(' '));
+            callback(...data.toString('utf8').split(' '));
```
Comment (ebolyen) [inline app/main.js]: That is so much nicer.

**Comment by thermokarst** [comment]
Woohoo! Thanks @jakereps, you rock! 🎉

**Pull request merged**